### ISSUE TITLE
VideoView.Dispose attaches to events instead of detaching

### ISSUE THREAD

**Issue by kimbirkelund** [Baseflow/XamarinMediaManager#648]
## 🐛 Bug Report

As the title says, the `VideoView.Dispose` method doesn't clean up event handlers, it creates more of them.

[From `VideoView.Dispose`:](https://github.com/Baseflow/XamarinMediaManager/blob/6a267f26989d9370882f4940d9c53b0aeacd5afb/MediaManager.Forms/VideoView.cs#L311-L314)

```csharp
MediaManager.Queue.QueueChanged += MediaQueue_QueueChanged;

MediaManager.PropertyChanged += MediaManager_PropertyChanged;
MediaManager.MediaPlayer.PropertyChanged += MediaPlayer_PropertyChanged;
```

**Comment by martijn00** [comment]
Fixed in 0.9.1

**Comment by kimbirkelund** [comment]
👍
